pkgs/development/python-modules/pwinput/default.nix

```diff
@@ -0,0 +1,27 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  lib,
+}:
+
+buildPythonPackage rec {
+  pname = "pwinput";
+  version = "1.0.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-yhqL0G4ohy11Hb1BMthjcSfCW0COo6NJN3MUpUkUJvM=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
+
+  doCheck = false;
```

Comment:
Please provide a reason in the comment.

Reply:
```
 # the tests fail, but the library works pretty well; has been tested with zotify
 ```
is that food?

Reply:
That's too long, just "Requires graphical environment to use pyautogui"

Reply:
Done

Reply:
Done.